src/CollectionService/index.php

```diff
@@ -0,0 +1,133 @@
+<?php
```

Comment:
This index.php could be removed as the `src/index.php` would be the one actually interacted with.